src/css/main.pre.css

```diff
@@ -11,11 +11,15 @@ body {
 }
 
 .card {
-  -webkit-animation: fadein 0.3s; /* Safari, Chrome and Opera > 12.1 */
-  -moz-animation: fadein 0.3s; /* Firefox < 16 */
-  -ms-animation: fadein 0.3s; /* Internet Explorer */
-  -o-animation: fadein 0.3s; /* Opera < 12.1 */
-  animation: fadein 0.3s;
```

Comment:
I removed this because I was getting a lot of flickering due to re-renders.  It looked really bad, but at some point it would be nice to put back in I think.